virtio-devices/src/vsock/unix/muxer.rs

```diff
@@ -480,55 +492,73 @@ impl VsockMuxer {
         }
     }
 
-    /// Parse a host "connect" command, and extract the destination vsock port.
-    ///
-    fn read_local_stream_port(
-        partial_command_map: &mut HashMap<RawFd, PartiallyReadCommand>,
-        stream: &mut UnixStream,
-    ) -> Result<u32> {
-        let command = partial_command_map.entry(stream.as_raw_fd()).or_default();
+    fn parse_port_from_read_command(command: &PartiallyReadCommand) -> Result<u32> {
+        // connect statement with a space, normally followed by the port and a `\n`
```

Comment:
```suggestion
        // normally followed by the port and a `\n`
```

We can see that it's a connect statement with a space from the string literal. :)